DMCompiler/DM/DMProc.cs

```diff
@@ -387,7 +387,7 @@ private class DMProcScope {
 
         public void CreateFilteredListEnumerator(DreamPath filterType) {
             if (!DMObjectTree.TryGetTypeId(filterType, out var filterTypeId)) {
-                DMCompiler.ForcedError($"Cannot filter enumeration by type {filterType}");
+                DMCompiler.Emit(WarningCode.ItemDoesntExist, Location, $"Cannot filter enumeration by type {filterType}");
```

Comment:
`Location` here refers to the location of the proc itself, and not the expression that called `CreateFilteredListEnumerator()`